gax-java/gax/src/test/java/com/google/api/gax/rpc/testing/FakeMtlsProvider.java

```diff
@@ -76,6 +58,11 @@ public KeyStore getKeyStore() throws IOException {
     return keyStore;
   }
 
+  @Override
+  public boolean isAvailable() throws IOException {
+    return true;
+  }
```

Comment:
qq, what is this used for? Was this added in by mistake?

Reply:
The updated MtlsProvider interface declares an "isAvailable()" method which is a new concept, but for the sake of testing and being backwards compatible with existing tests, the "FakeMtlsProvider" is "always available".

Reply:
Ah right. Thanks for clarifying.